src/model/model.cpp

```diff
@@ -315,7 +315,21 @@ int Model :: compute_radiation_field_shortchar_order_0 ()
 
     Solver solver;
     solver.setup <CoMoving>        (*this);
-    solver.solve_shortchar_order_0 (*this);
+    if (parameters->one_line_approximation)
+    {
+        solver.solve_shortchar_order_0 <OneLine> (*this);
+    }
+    else
+    {
+        if (parameters->sum_opacity_emissivity_over_all_lines)
+        {
+            solver.solve_shortchar_order_0 <None> (*this);
+        }
+        else
+        {
+            solver.solve_shortchar_order_0 <CloseLines> (*this);
+        }
+    }
```

Comment:
Not a big fan of the nested ifs.
Wouldn't it be better to have separate if clauses that include the return statement, to avoid nesting?